services/storage-users/pkg/command/uploads.go

```diff
@@ -69,7 +73,7 @@ func ListUploads(cfg *config.Config) *cli.Command {
 			fmt.Println("Incomplete uploads:")
 			for _, u := range uploads {
 				ref := u.Reference()
-				fmt.Printf(" - %s (Space: %s, Name: %s, Size: %d/%d, Expires: %s, Processing: %t)\n", ref.GetResourceId().GetSpaceId(), u.ID(), u.Filename(), u.Offset(), u.Size(), u.Expires(), u.IsProcessing())
+				fmt.Printf(" - %s (Space: %s, Name: %s, Size: %d/%d, Expires: %s, Processing: %t)\n", u.ID(), ref.GetResourceId().GetSpaceId(), u.Filename(), u.Offset(), u.Size(), u.Expires(), u.IsProcessing())
```

Comment:
wtf how did this ever work? if @wkloucek is relying on the output of the `ocis storage-users uploads list` command he should have run into this ...

Reply:
Please lets change this. Both are uuids, you cannot see this bug when using the commands. You need to look closely into the code. Only ocis logs will complain.

Reply:
ok, but please make an explcit note in the changelog that the output changed